lib/govuk_publishing_components/presenters/step_by_step_nav_helper.rb

```diff
@@ -46,6 +46,7 @@ def list(element)
                 :li,
                 class: "gem-c-step-nav__list-item js-list-item #{link_active(contents[:active])}",
               ) do
+                @options[:ga4_index_total] = element[:contents].length
```

Comment:
Does this need a `if @options[:ga4_tracking]` at the end?

Reply:
@andysellick Good spot, done.